lib/neo4j/active_node/scope.rb

```diff
@@ -45,6 +45,10 @@ def scope(name, proc)
             _call_scope_context(eval_context, query_params, proc)
           end
         end
+
+        define_method(name) do |*query_params|
+          as(:n).send(name, *query_params)
```

Comment:
not that it really matters, but perhaps this should be `public_send`

Reply:
Okay Sure.